db/migrate/20190504135710_add_user_id_to_records.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToRecords < ActiveRecord::Migration[5.2]
+  def change
+    add_column :records, :user_id, :integer
```

Comment:
Rails 5+ has belongs_to type like

```suggestion
    change_table :records do |t|
      t.belongs_to :user, index: true, null: false
    end
```